sfaira/commands/templates/multiple_datasets/cookiecutter.json

```diff
@@ -13,11 +13,13 @@
     "normalization": "",
     "organ": "",
     "organism": "",
-    "assay": "",
+    "assay_sc": "",
```

Comment:
shouldnt this file also have `doi_journal` etc rather than doi?

Reply:
The doi is also used for the naming of the file. By default the doi is then used as the doi_journal.